app/build.gradle.kts

```
@@ -2,6 +2,10 @@ plugins {
     alias(libs.plugins.android.application)
     alias(libs.plugins.kotlin.android)
     alias(libs.plugins.kotlin.compose)
+    alias(libs.plugins.google.gms.google.services)
+    alias(libs.plugins.google.firebase.crashlytics)
+    id("com.google.firebase.firebase-perf")
```

add it to `plugins` section in `libs.version.toml` and use it like this 
```suggestion
    alias(libs.plugins.google.firebase.firebase-perf)
```

---

done